test-runner-frontend/src/Views/ReportUtils.js

```diff
@@ -0,0 +1,86 @@
+import html2canvas from 'html2canvas';
+import jsPdf from 'jspdf';
+
+export const builChartsData = (from) => {
+  const classesAndBenchmarks = {};
+  from?.statistics?.map((testRun, index) => {
```

Comment:
can we please change `.map` to `.forEach` as we're not using the returned data

Reply:
Done.